clang/lib/Basic/SourceManager.cpp

```diff
@@ -46,6 +47,12 @@ using namespace clang;
 using namespace SrcMgr;
 using llvm::MemoryBuffer;
 
+#define DEBUG_TYPE "source-manager"
+
+STATISTIC(
+    MaxUsedSLocBytes,
+    "Maximum number of bytes used by source locations (both loaded and local)");
```

Comment:
nit: It may also be helpful to add some background on why this is important to track.

Reply:
Done. I added a comment because the description string is something that tends to rather look like a one-liner.